lib/features/popup-menu/PopupMenu.js

```diff
@@ -262,6 +262,8 @@ PopupMenu.prototype.close = function() {
 
   this.reset();
 
+  this._canvas.focus();
```

Comment:
Proposal: Consider if an element other than the document was focused, and if not, restore canvas focus.

Rationale: A popup menu action _may_, i.e. open a modal, and we don't want to take the focus if the modal was focused.

TODO: Also write a test case for such a scenario.

Reply:
Done 👍

Do you have an idea how to mock menu closing into some modal-like container?

Reply:
Create a DOM element outside of the Canvas. Focus it as part of the popup menu action. Observe the result.

Reply:
I don't think this is relevant anymore since this logic was moved to `Canvas`.